guide/getting-started/hardware.md

```diff
@@ -145,6 +147,26 @@ Bitcoin ATMs are usually bound to local money transmission laws and regulations,
 
 You can find local ATMs using services like [coinatmradar](https://coinatmradar.com/).
 
+## Point of sale
+
+People looking to accept bitcoin payments, especially when using the lightning network, sometimes use dedicated point of sale (POS) terminals when selling goods and services.
+
+{% include picture.html
+   image = "/assets/images/guide/getting-started/hardware/point-of-sale.jpg"
+   retina = "/assets/images/guide/getting-started/hardware/point-of-sale@2x.jpg"
+   mobile = "/assets/images/guide/getting-started/hardware/point-of-sale-mobile.jpg"
+   mobileRetina = "/assets/images/guide/getting-started/hardware/point-of-sale-mobile@2x.jpg"
+   alt-text = "Examples of point of sale hardware devices."
+   width = 800
+   height = 400
+%}
```

Comment:
Would it be a good idea to include a tablet in this image as well? Something like one of the Square POS tablets mounted on the swivel? While the tiny little brick device is definitely a thing, many businesses accepting bitcoin are doing it through a tablet based POS and I only see that trend continuing.

I propose this in the same spirit as the hardware wallets image which shows devices of all shapes and sizes.

Reply:
Yeah I can add that, good suggestion!